### ISSUE TITLE
clothes

### ISSUE THREAD

**Issue by ShynuSj** [renzuzu/renzu_clothes#33]
hi  renzu , really appreciate your work,, 
    i need to know that ,can we merge the different categorized blips into one or few.. inorder to reduce the blips..???

**Comment by renzuzu** [comment]
yes its possible, im also planning to merge the blips, thanks for reminding
